libraries/botbuilder-ai-orchestrator/src/orchestratorComponentRegistration.ts

```diff
@@ -12,14 +12,25 @@ import { BuilderRegistration, ComponentRegistration, ResourceExplorer } from 'bo
 
 import { OrchestratorAdaptiveRecognizer } from './orchestratorAdaptiveRecognizer';
 
+/**
+ * Define component assets for Luis.
+ */
 export class OrchestratorComponentRegistration implements ComponentRegistration {
     private readonly _builderRegistrations: BuilderRegistration[] = [];
     private _resourceExplorer: ResourceExplorer;
 
+    /**
+     * Gets all the builder registrations instances.
```

Comment:
I think the plural here goes only to instances, but I'm not sure.
```suggestion
     * Gets all the builder registration instances.
```

Reply:
I think `registrations` its ok.
Maybe @ceciliaavila or @iscaltritti-sw can apport on this.

Reply:
I agree with @santgr11, hard to explain grammatically. 
Registrations instances would be correct for a matrix where each builder gets more than one registration.